File: 58.suma_datos/suma_datos.py
# Ejercicio No. 58: Suma de datos.

# processing
def sumar_datos(lista):
    suma = 0
    for i in lista:
        suma += i
    return suma

lista1 = [2,3,4,5,6]
suma = sumar_datos(lista1)

# output
print(suma)

File: 58.suma_datos/SumaDatos.java
import javax.swing.JOptionPane;

public class SumaDatos{
    //Processing
    public static int sumarDatos(int[] lista){
        int suma = 0;
        for(int i = 0; i<lista.length; i++){
            suma += lista[i];
        }
        return suma;
    }
    public static void main(String[] args) {
        int[] lista1 ={2, 3, 4, 5, 6};
        int suma = sumarDatos(lista1);

        //Output
        JOptionPane.showMessageDialog(null, suma);
        
        System.exit(0);
    }


}